agent/command_test.go

```diff
@@ -145,10 +145,10 @@ func (s *CommandSuite) TestShellExec() {
 	nextTask := &apimodels.NextTaskResponse{
 		TaskId:     s.tc.task.ID,
 		TaskSecret: s.tc.task.Secret,
-		TaskGroup:  s.tc.taskGroup,
+		TaskGroup:  "",
 	}
 	shouldSetupGroup := !s.tc.ranSetupGroup
-	taskDirectory := s.tc.taskDirectory
+	taskDirectory := ""
```

Comment:
If `s.tc.taskDirectory` supposed to be empty? It seems like for the agent suite, it pre-initializes a temporary directory for the task to use, but this test suite does not. This `s.tc` likely needs to have a temporary directory set as well so it can get cleaned up after the test finishes.

Reply:
this is no longer necessary, I reset it